### DISCUSSION TITLE
1.6.0 버전 카메라가 좀 불편해요

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#1221] [4. General]
1.6.0 버전이 나와서 한 번 사용을 해봤는데 카메라가 좀 안 좋게 바뀐 것 같아요.

예전에는 벽에 닿으면 카메라가 최대한 가까이 오면서 캐릭터의 모습을 놓치지 않았는데

이번에 바뀐 것은 벽에 닿은 상태에서 카메라 회전을 하면 캐릭터 자체가 컬링이 되어서 좀 이상해요.

**Comment by Ted-0323** [comment]
카메라와 캐릭터가 붙었을 경우 캐릭터 자체가 컬링이 되어 보이지 않는 [해당 공지](https://github.com/naverz/zepeto-studio-kor/discussions/1183)의 Enhanced ZEPETO Camera Collision 이미지와 같은 기능을 말씀하시는 거라면 하이어라키에서 ZepetoPlayers를 선택하신 후 Collision의 Use Character Culling을 끄시면 됩니다.
이와 다르게 저는 카메라가 자꾸 특정 조건에서 벽을 뚫어버리네요... 머리가 아픕니다 ㅜㅜ

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 정확한 문제 파악을 위해 어떤 조건에서 벽을 뚫어버리는지 설명 부탁드립니다.

**Reply by Ted-0323** [reply]
영상으로 보시면 좋을 것 같습니다. [유튜브](https://youtu.be/waBDedGEaXM)에 업로드했습니다.
예전 카메라는 벽의 거리와 상관없이 내 카메라와 캐릭터 사이에 오브젝트가 존재하면 무조건 막혔었는데, 지금은 영상에 나와있다시피 오브젝트의 두께에 따라 뚫릴지 안뚫릴지가 달라집니다.

또한 뒷부분은 '중력에 의해 자연스럽게 지형을 내려갑니다' 패치 이후로 보이는 문제인데, 45도 각도의 벽을 타고 내려갈 때에 캐릭터가 덜덜 떨면서 내려오게 됩니다. 현재 사용중인 1.4.0 버전과는 조금 다른 모습을 보여줘서 같이 올려뒀습니다. 참고로  [1.4.0에서 같은 조건으로 테스트할 시 덜덜 떨리는 문제는 없습니다.](https://youtu.be/q-LLb2bl8vM)

**Reply by Zepeto-TechEvangelist** [reply]
상세한 영상 설명 감사 합니다. 내부에서 검토 후 안내 드리겠습니다.

**Reply by Ted-0323** [reply]
아 그리고 혹시 1.4.0에서 Character culling 기능만 사용하는 방법은 없을까요? 구버전을 계속 사용하고 싶은데 Culling 기능만 따로 사용하고 싶습니다

**Reply by Zepeto-TechEvangelist** [reply]
구 버전 카메라에 컬링 추가하는 부분은 지원이 어렵습니다. 오브젝트 두께에 따른 충돌 예외 상황에 대해서 개선하는 방향으로 준비 중입니다.
감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 질문주신 컬링 부분은 위 답변자님이 말씀해주신 것처럼 Use Character Culling 옵션으로 제어가 가능합니다.
https://studio.zepeto.me/kr/guides/character-control 
가이드 하단에 각 항목 설명 참고 부탁 드립니다.
